pkg/cableengine/healthchecker/healthchecker.go

```diff
@@ -16,121 +16,116 @@ type LatencyInfo struct {
 }
 
 type Interface interface {
-	Start(stopCh <-chan struct{})
+	Start(stopCh <-chan struct{}) error
 
-	GetLatencyInfo(hostName string) *LatencyInfo
+	GetLatencyInfo(hostName string) interface{}
```

Comment:
Why was this changed from a concrete type to `interface{}`?

Reply:
it was an oversight , when I was fixing some compilation issues reverted it.